monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```
@@ -765,23 +765,32 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
       OsdDeviceDetailsPanel(
         '$osd R/W Bytes', '', 'bytes', 'Read (-) / Write (+)', 'irate(ceph_osd_op_r_out_bytes{ceph_daemon=~\"$osd\"}[1m])', 'irate(ceph_osd_op_w_in_bytes{ceph_daemon=~\"$osd\"}[1m])', 'Read Bytes', 'Write Bytes', 12, 1, 6, 9)
       .addSeriesOverride({"alias": "Read Bytes","transform": "negative-Y"}),
-      addRowSchema(false, true, 'Physical Device Performance') + {gridPos: {x: 0, y: 10, w: 24, h: 1}},
+      addPieChartSchema({}, '$datasource', '', 'Under graph', 'pie', '$osd onode Hit, Miss Requests', 'current')
+      .addTarget(
+        addTargetSchema('ceph_bluestore_bluestore_onode_hits{ceph_daemon=~\"$osd\"}', 1, 'time_series', '$osd hits')
```

No need to escape double quotes. That's inherited from the JSON code, but jsonnet supports both double and single quotes, which is the one you're using here:
```suggestion
        addTargetSchema('ceph_bluestore_bluestore_onode_hits{ceph_daemon=~"$osd"}', 1, 'time_series', '$osd hits')
```

---

Done. Thanks @epuertat